coral/config/genesis.json

```diff
@@ -52,6 +52,25 @@
             "account_number": "0",
             "sequence": "0"
           }
+        },
+        {
+          "type": "cosmos-sdk/Account",
+          "value": {
+          "address": "cosmos1xgf0cm289z0z2w5phhy6ay3xscq5fpmt8xefhn",
```

Comment:
Thank you @KamiD!

I'm not sure how to generate them, but this address must have a `coral` prefix.

Also, it would be good to check indent on the JSON 💅

Reply:
I use command `coral keys add` to generating them, it seems always have `cosmos` prefix in `address`.
when i use coral keys add to generate a new key, the key list is looks like this:
```sh
./coral keys list
Enter keyring passphrase:
- name: val-okchain
  type: local
  address: cosmos1xgf0cm289z0z2w5phhy6ay3xscq5fpmt8xefhn
  pubkey: coralpub1addwnpepqfe7fnmeqwcvpcafvrksqn6cm2q2r9wh8nwmc6q5az8w9ey63tw450fws0c
  mnemonic: ""
  threshold: 0
  pubkeys: []
```

I am using tag of v0.10.0 to build this bin, building command like:
```sh
make build-coral
```

Reply:
```sh
coral version
0.10.0
```

Reply:
updating go version to 1.14.6, problem resolved